src/registrar/models/domain_application.py

```diff
@@ -409,6 +409,7 @@ class BranchChoices(models.TextChoices):
     )
 
     federal_agency = models.TextField(
+        choices=AGENCY_CHOICES,
```

Comment:
**question** what is the benefit of keeping this as a textfield and not a charfield like it is below?

Reply:
I kept it as a text field since that is what is waqs originally.  Text fields are typically larger than char fields, so truncating it down to a char field (while probably ok) was an assumption I decided not to make.  But, if we all agree that the dropdown menu options will generally be short enough (I think they *should* be short enough), then char would be cleaner.

Reply:
I would think a char fields length is determine by the max_length parameter, which defaults to 255 which should be more than enough for a drop down option. How it changes the display, I can't recall. It's up to you really if you want to change this, I'm not hard set either way